### PR TITLE
Update to Bootstrap v4.3.1

### PR DESCRIPTION
**Pull request by dennisholmer**
Solves issue #7

### CODE REVIEW

**Comment by dennisholmer** [comment]
~~This contains breaking changes to all projects that still have BS 3.~~

File: projects/cron-editor/src/lib/cron-editor.component.css [hunk 1]
```diff
@@ -1,69 +1,23 @@
-.cron-editor-container {
-    margin-top: 10px;
+tab-pane .well,
```
Comment (dennisholmer) [inline projects/cron-editor/src/lib/cron-editor.component.css]:
```suggestion
.tab-pane .well, 
```

**Review by davidst — CHANGES_REQUESTED** [review]
I see that you changed the formatting from 4 spaces to 2 spaces. This makes the diff very hard to follow and is also inconsistent with the rest of the files in the project. Can you please go back to 4 spaces formatting?

**Comment by dennisholmer** [comment]
Uhm, was this better? It was inconsistent before I made changes. Should TS have 2 and HTML/CSS have 4 or should all have 4? As you can see I updated the .editorconfig and tslint.json to 4 spaces.

**Comment by andrewryan1906** [comment]
Hey, any movement on this? I'm assuming until this is done, this component won't work with Bootstrap 4?
